Art/thiziri_bouaziz/styles.css

```diff
@@ -0,0 +1,134 @@
+body{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    position: relative;
+    display: flex;
+    justify-content: center;
+}
+
+.container{
+width: 100%;
+height: 100%;
+display: flex;
+justify-content: center;
+text-align: center;
+
+}
+
+
+body{
+        -webkit-animation: color-change-2x 2s linear infinite alternate both;
+                animation: color-change-2x 2s linear infinite alternate both;
+    }
+ 
+@-webkit-keyframes body {
+  0% {
+    background: #19dcea;
+  }
+  100% {
+    background: #b22cff;
+  }
+}
+@keyframes color-change-2x {
+  0% {
+    background: #19dcea;
+  }
+  100% {
+    background: #b22cff;
+  }
+}
+
+h1{
+    padding: 30px;
+    border-radius: 40px;
+    background-color: rgba(211, 10, 167, 0);
+    color: rgb(79, 12, 100);
+    /* position: absolute; */
```

Comment:
## Unnecessary comment
Please remove the commented line

Reply:
yes i remove it